CovertInfiltration/Config/XComGameBoard.ini

```diff
@@ -251,6 +251,61 @@ MaxActionHours[2]=168
 MaxActionHours[3]=240
 
 [CovertAction_TechRush X2CovertActionTemplate]
+MinActionHours[0]=120
+MinActionHours[1]=120
+MinActionHours[2]=120
+MinActionHours[3]=168
```

Comment:
Why did you reduce the duration of the Inspire Tech CA? It was fine before, it should be the same duration as the Breakthrough Tech CA from vanilla.